packages/core/src/ceramic.ts

```diff
@@ -205,7 +205,6 @@ export class Ceramic implements CeramicApi {
     this._networkOptions = params.networkOptions
     this._loadOptsOverride = params.loadOptsOverride
     this._indexing = modules.indexing
```

Comment:
Related to my comment on the previous PR: now that we pass the indexing into the IndexApi, do we need to store a reference here in the Ceramic class, or can we just keep the reference to `this.index` and have it be responsible for managing the `DatabaseIndexApi`?